pkgs/by-name/xw/xwayland-satellite/package.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, clang
+, xwayland
+, xorg
+, xcb-util-cursor
+, libxcb
+, llvmPackages
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "xwayland-satellite";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+    owner = "Supreeeme";
+    repo = "xwayland-satellite";
+    rev = "v${version}";
+    hash = "sha256-dwF9nI54a6Fo9XU5s4qmvMXSgCid3YQVGxch00qEMvI=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
```

Can be simplified as xwayland-satellite does not use git dependencies:
```suggestion
  cargoHash = "<whatever>";
```